src/modules/collection/sagas.ts

```diff
@@ -420,6 +410,31 @@ function* handleTransactionSuccess(action: FetchTransactionSuccessAction) {
   }
 }
 
+/**
+ * Post proccesses a collection that was published to the blockchain by singaling the
+ * builder server that the collecton has been published, setting the item ids,
+ * deploys the item entities to the Catalyst sever and creates the forum post.
+ *
+ * @param collection - The collection to post process.
+ */
+function* postProcessCollectionTransaction(collection: Collection) {
```

Comment:
Do you think we can figure out a concept for this method's name so we can avoid calling it `postProcess`?  I was thinking `finishCollectionPublish` or `complete(Collection)Publish` or something similar?

What do you think?

Reply:
Changed it to `finishCollectionPublishing`